ui/src/alerting/components/CheckEOHeader.tsx

```diff
@@ -64,7 +103,11 @@ const CheckEOHeader: FC<Props> = ({name, onSetName, onCancel, onSave}) => {
         />
       </Page.Header.Left>
       <Page.Header.Center widthPixels={300}>
-        <CheckAlertingButton />
+        <CheckAlertingButton
```

Comment:
What was your motivation for disconnecting it from Redux? We've moved this button around a bunch and it was nice to not worry about reconnecting the right props every time. I bet we'll end up moving it again since I think it makes more sense to be closer to the query/alert builder, which is the part of the UI that changes when you click it.

Reply:
Two sibling both required `draftQueries` to make independent display decisions.  Rather than connect two  children to redux, I decided to pull the state up to their shared parent and pass props down one level.  Mayhaps I'll regret this decision.